grpc-ballerina/tests/resources/generated-sources/tool_test_unary_9/helloWorldString_pb.bal

```diff
@@ -0,0 +1,90 @@
+import ballerina/grpc;
```

Comment:
License header is missing ?

Reply:
These files use to test the generated code. Since the generated code does not have any license we do not add it here.